common/src/models/resource.rs

```diff
@@ -159,3 +167,32 @@ fn get_persist_table(persist_instances: &[&Response], service_name: &str) -> Str
         service_name
     )
 }
+
+fn get_custom_resources_table(
+    custom_resource_instances: &[&Response],
+    service_name: &str,
+) -> String {
+    let mut table = Table::new();
+
+    table
+        .load_preset(UTF8_FULL)
+        .apply_modifier(UTF8_ROUND_CORNERS)
+        .set_header(vec![Cell::new("Instances")
+            .set_alignment(CellAlignment::Center)
+            .add_attribute(Attribute::Bold)]);
+
+    for (idx, _) in custom_resource_instances.iter().enumerate() {
+        // TODO: add some information that would make the custom resources identifiable.
+        // This requires changing the backend resource list response to include a resource identifier
+        // that can be used to query for more info related to a resource.
```

Comment:
I think we can just print some JSON output for now

Reply:
I am not sure what that would be. We can print the `config` or the `data`. Some of them are subject to leaking secrets. I would keep this as is for now and we'll see how this can be improved later on. What do you think?

Reply:
Yip, sounds good to change it once we have a better idea of what to print